src/main/resources/openapi/transcriptions.yaml

```diff
@@ -437,6 +437,9 @@ components:
           type: string
           format: date-time
           example: '2023-07-31T14:32:24.000Z'
+        is_manual:
```

Comment:
I'm not sure why this was added to the transcriptions api endpoint?
Pretty sure I asked this in a refinement the other week and @jackmaloney didn't want the request body to be changed as the idea was that all users using this endpoint would be manual requests as the automatic requests just use the underlying service instead.

Reply:
Ah didn't know where the auto was coming from, e.g. through the API or an internal job